src/routes/countries.js

```
@@ -0,0 +1,8 @@
+const express = require('express');
```

ESLint

[error] Extra semicolon.

(semi)